modules/lss-missionKeyword/missionKeyword.user.js

```diff
@@ -1282,189 +1585,194 @@
     var color_thw      = '#03a8f9';
     var color_sonst    = '#03a8f9';
 
-	//Stichwort Element
-    var title_stichwort = document.getElementById('missionH1');
+	if(I18n.locale == 'de')
+		default_aao = default_aao_de;
+	else if(I18n.locale == 'en')
+		default_aao = default_aao_en;
+	else if(I18n.locale == 'nl')
+		default_aao = default_aao_nl;
+	else
+		default_aao = default_aao_de;
+		
 	
-	//Anzahl Patientenbalken
-    var anzahl_patients = document.getElementsByClassName("patient_progress").length;
-    var aao_text = '';
+	$.each(default_aao, function(key, val) {
+		var settingsPropertyName = 'missionKeyword-'+key;
+		var tmpObject = {
+			[settingsPropertyName] : {
+				"default": val,
+				"ui": {
+					"label": I18n.t('lssm.missionKeywordDefaults.'+key)+ " (ID "+key+")",
```

Comment:
sollte normal nie passieren, denn es gibt in `default_aao` genausoviele Einsatz-IDs pro Sprache wie es in der `lssm.missionKeywordDefaults.keywords` gibt